config/solrconfig.xml

```diff
@@ -35,7 +35,7 @@
        that you fully re-index after changing this setting as it can
        affect both how text is indexed and queried.
   -->
-  <luceneMatchVersion>8.1.1</luceneMatchVersion>
+  <luceneMatchVersion>9.7</luceneMatchVersion>
```

Comment:
Solr is apparently already installing 9.7 anyway, but this just puts the config in line with that fact.